std/haxe/io/FPHelper.hx

```diff
@@ -54,8 +54,83 @@ class FPHelper {
 		}
 	#elseif js
 		static var helper = new js.html.DataView(new js.html.ArrayBuffer(8));
-	#else
+	#end
+
+	#if !(neko || cpp || cs || java || flash || (js && nodejs))
 		static inline var LN2 = 0.6931471805599453; // Math.log(2)
+
+		static inline function i2f(i: Int): Float {
+			var sign = 0x80000000 & i == 0 ? 1.0 : -1.0;
+			var e = (i >> 23) & 0xff;
+			if (e == 255)
+				return i & 0x7fffff == 0
+					? (sign > 0 ? Math.POSITIVE_INFINITY : Math.NEGATIVE_INFINITY)
+					: Math.NaN;
+			var m = e == 0 ? (i & 0x7fffff) << 1 : (i & 0x7fffff) | 0x800000;
+			return sign * m * Math.pow(2, e - 150);
+		}
+
+		static inline function ii2d(lo: Int, hi: Int): Float {
```

Comment:
Please, rename these methods to something human-readable

Reply:
So, which names are appropriate? :sweat_smile: `i64_to_double/double_to_i64`, `i32_to_float/float_to_i32` or something else?

Reply:
Like that, but with camel case )

Reply:
These names `i32ToFloat/floatToI32/i64ToDouble/doubleToI64` are already exists,  I can only get the following:
```
i2f, f2i, i2f64, f2i64,

itof32, ftoi32, itof64, ftoi64,
		
i32ToF32, f32ToI32, i64ToF64, f64ToI64

_i32ToFloat, _floatToi32, _i64ToDouble, _doubleToI64
```
Any suggestions?

Reply:
The latter option looks best to me.